### ISSUE TITLE
Keep workspace loaded option per workspace

### ISSUE THREAD

**Issue by tobigue** [kytwb/ferdi#37]
I love the possibility to have workspaces that are not loaded.
But it would be great to be able to configure this **per workspace**, so that one can have multiple loaded and multiple inactive workspaces.
:heart:

**Comment by vantezzen** [comment]
Great idea. We would have to see where we could save this information and how we could sync it between clients as the Franz server won't save any settings the original Franz client doesn't have.

I have created a new branch "issue-37" to implement this.

Related: <https://github.com/kytwb/ferdi/blob/master/src/stores/ServicesStore.js#L122> is where Ferdi decides which Workspaces to keep loaded.

**Comment by kytwb** [comment]
@all-contributors please add @tobigue for ideas

**Comment by vantezzen** [comment]
I have now implemented your feature in the issue-37 branch and it should be available in the next update.

I also found a way to save the setting to the Franz API (by adding a "secret", non-existent service to the workspace). 
The setting should now sync between Ferdi clients while still being fully compatible with Franz clients as they will simply ignore it. 
The only downside with this method is, that the setting will be reset if you update (change the services contained in the workspace) through a Franz client as Franz won't add the "secret", non-existent service.
